build.gradle

```diff
@@ -1,11 +1,12 @@
-buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:0.6'
-    }
+plugins {
+    id 'groovy'
+    id 'idea'
+    id 'maven-publish'
+    id 'java-gradle-plugin'
+    id 'com.jfrog.bintray'              version '1.8.4'
+    id 'net.nemerosa.versioning'        version '2.7.1'
```

Comment:
I personally won't use these two plugins. Let's remove them for now.

Reply:
OK